invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -42,14 +44,21 @@ def MChits_and_detsim_electron_simulation_params(nexusMC_filename):
     transverse_diffusion   = 1.00 * units.mm / units.cm**0.5
     longitudinal_diffusion = 0.30 * units.mm / units.cm**0.5
 
-    return xs, ys, zs, energies,\
-           wi, fano_factor, lifetime, drift_velocity,\
-           transverse_diffusion, longitudinal_diffusion
+
+    typename = "MChits_and_detsim_params"
+    field_name = ["xs", "ys", "zs", "energies", "wi", "fano_factor", "lifetime",
+                  "drift_velocity", "transverse_diffusion", "longitudinal_diffusion"]
+
+    nt = namedtuple(typename, field_name)
+    return nt(xs, ys, zs, energies, wi, fano_factor, lifetime,
+              drift_velocity, transverse_diffusion, longitudinal_diffusion)
```

Comment:
To increase safety, you could consider keyword args: `return nt(xs=xs, ys=ys ...)`. Tedious, but safe.

To alleviate the tedium you could do something like

```python

In [1]: from collections import namedtuple                                                                                                                                                                                                                                        

In [2]: from operator import itemgetter                                                                                                                                                                                                                                           

In [3]: one, two, three = 10,20,30    # Replace with the actual variables you use                                                                                                                                                                                                                                  

In [4]: names = 'one two three'                                                                                                                                                                                                                                                   

In [5]: XXX = namedtuple('XXX', names)                                                                                                                                                                                                                                            

In [6]: XXX(*itemgetter(*XXX._fields)(locals()))                                                                                                                                                                                                                                  
Out[6]: XXX(one=10, two=20, three=30)
```

which is more cryptic, but is quite DRY and thus makes it difficult to mess up the order, now or in the future.